src/web/layouts/CommentLayout.tsx

```diff
@@ -235,25 +261,42 @@ export const CommentLayout = ({ CAPI, NAV }: Props) => {
                     </GridItem>
                     <GridItem area="headline">
                         <div className={maxWidth}>
-                            <ArticleHeadlinePadding
-                                designType={CAPI.designType}
-                            >
-                                <ArticleHeadline
-                                    headlineString={CAPI.headline}
-                                    designType={CAPI.designType}
-                                    pillar={CAPI.pillar}
-                                    webPublicationDate={CAPI.webPublicationDate}
-                                    tags={CAPI.tags}
-                                    byline={CAPI.author.byline}
-                                />
-                            </ArticleHeadlinePadding>
+                            <div className={avatarHeadlineWrapper(!!avatarUrl)}>
```

Comment:
take a step back why not use `cx(baseStyles, hasAvatar ? minHeightStyles : '')` ?

Reply:
That's a really good suggestion, this makes the code much more readable.

I stuck with the && pattern as that's the pattern used elsewhere but 👍 for seeing this